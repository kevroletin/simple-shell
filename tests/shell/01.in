echo 123
echo hello
echo hello.world
