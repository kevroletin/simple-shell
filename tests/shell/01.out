123
hello
hello.world
